microsoft-edge/webview2/how-to/print.md

```diff
@@ -0,0 +1,793 @@
+
+##### [Win32/C++](#tab/win32cpp)
+
+```cpp
+bool AppWindow::ShowPrintUI(COREWEBVIEW2_PRINT_DIALOG_KIND printDialogKind)
+{
+  wil::com_ptr<ICoreWebView2_16> webView2_16;
+  CHECK_FAILURE(m_webView->QueryInterface(IID_PPV_ARGS(&webView2_16)));
+  CHECK_FAILURE(webView2_16->ShowPrintUI(printDialogKind));
+  return true;
+}
+```
+
+**APIs:**
+
+* [ICoreWebView2_16::ShowPrintUI method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#showprintui)
+* [COREWEBVIEW2_PRINT_DIALOG_KIND enum](/microsoft-edge/webview2/reference/win32/icorewebview2_16#corewebview2_print_dialog_kind)
+
+---
+
+<!-- end of tab-set -->
+
+
+<!-- ====================================================================== -->
+## The Print method to customize printing
+
+The `Print` method silently prints the current top-level document in the WebView2 control by using optional, programmatically specified print settings. If you want to build your own print preview dialog, or build your own print experience, you can use this method. This API consists of an asynchronous `Print` method and a `PrintSettings` object.
+
+
+<!-- ------------------------------ -->
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2.PrintAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printasync#microsoft-web-webview2-core-corewebview2-printasync)
+* [CoreWebView2Environment.CreatePrintSettings Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createprintsettings)
+* [CoreWebView2PrintSettings Class](/dotnet/api/microsoft.web.webview2.core.corewebview2printsettings)
+* [CoreWebView2PrintStatus Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2printstatus)
+
+<!-- ------------------------------ -->
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2.PrintAsync Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#PrintAsync)
+* [CoreWebView2Environment.CreatePrintSettings Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createprintsettings)
+* [CoreWebView2PrintSettings Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2printsettings)
+* [CoreWebView2PrintStatus Enum](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2printstatus)
+
+
+<!-- ------------------------------ -->
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2_16::Print method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#print)
+* [ICoreWebView2Environment6::CreatePrintSettings method](/microsoft-edge/webview2/reference/win32/icorewebview2environment6#createprintsettings)
+* [ICoreWebView2PrintSettings2 interface](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2)
+* [COREWEBVIEW2_PRINT_STATUS enum](/microsoft-edge/webview2/reference/win32/icorewebview2_16#corewebview2_print_status)
+
+---
+
+<!-- end of tab-set -->
+
+#### Example 1: The Print method to customize printing
```

Comment:
consider differentiating Example 1 vs 2 in headings.

current design:
```
## The Print method to customize printing
#### Example 1: The Print method to customize printing
#### Example 2: The Print method to customize printing
```

Example 1 does:  
This example prints the current web page to the default printer, using the default print settings, without opening a **Print** dialog.

Example 2 does:
This example prints the current webpage to a specific printer, using the specified settings.

How to hint at the difference, using a few words only, in the h4 headings?